normative/dtpr-stickers
# Redesigns sensor page

# Notion card link(s)
- [Refactor Sensor Details](https://www.notion.so/normative/Refactor-Sensor-Detail-58267681d6e94c399edd3d72da604c9f)

# Review APP
- [Heroku Review App](https://dtpr-sticker-chore-sens-inf5iq.herokuapp.com/sensors/myd2vKdzrsn7RaF4t_UIb)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Adds sensor tabs for **Details** and **FAQ**
- [x] Adds FAQ tab content
- [x] Refactors sensor page completely
- [x] Refactors and redesigns emojis Feedback
- [x] Refactors and redesigns Feedback session
- [x] Amends Footer styles
- [x] Adds GA tracking to Feedback responses
- [x] Adds sensor name to SensorAccordion track event
- [x] Adds sensor place section

# How to test it
- Run into your machine, navigate to `/sensors/<sensorId>`
- Check if page is still working and designed as [Figma](https://www.figma.com/file/t1ZIjYmejw3bNc9hxMNzGB/Helpful-Places?node-id=91%3A466)
